platform/pulp/platform/models/repository.py

```diff
@@ -214,7 +214,6 @@ class Importer(Plugin):
     scratchpad = fields.GenericRelation(Scratchpad)
 
     class Meta:
-        abstract = True
         unique_together = ('name', 'repository')
```

Comment:
Just noticed that the `unique_together` should not be on the `Importer.Meta`.  These models went through several revisions and this `unique_together` was moved to `RepositoryImporter` along with the FK but I missed removing it here.  Please remove it.